qingstor/sdk/client/remote_encryption_client.py

```diff
@@ -0,0 +1,32 @@
+import base64
+import hashlib
+
+from Crypto.Cipher import AES
+
+
+class RemoteEncryptionClient:
+
+    def __init__(self,bucket,encrypt_key,encrypt_algo="AES256"):
+        self.bucket=bucket
+        self.encrypt_key=encrypt_key
+        self.encrypt_algo=encrypt_algo
+        self.encoded_encrypt_key = base64.b64encode(self.encrypt_key)
+        self.encrypt_key_md5=self.cal_md5(self.encrypt_key)
+
+    def put_object(self,*args,**kwargs):
+        md5_dict_key="x_qs_encryption_customer_key_md5"
+        if  md5_dict_key not in kwargs:
+            encrypt_key_md5=self.encrypt_key_md5
+            encoded_encrypt_key_md5 = base64.b64encode(encrypt_key_md5)
```

Comment:
This could be done in `__init__`, we don't need to do it every time we put_object.